tunnel.go

```diff
@@ -56,40 +71,51 @@ func NewTunnel(account *nkn.Account, identifier, from, to string, tuna bool, con
 		return nil, ErrUDPNotSupported
 	}
 
-	fromNKN := len(from) == 0 || strings.ToLower(from) == "nkn"
-	toNKN := !strings.Contains(to, ":")
+	udpConnExpired := cache.NoExpiration
+	if config.UDPIdleTime > 0 {
+		udpConnExpired = time.Duration(config.UDPIdleTime) * time.Second
+	}
+
+	fromNKN := false
+	for _, f := range from {
+		fromNKN = (len(f) == 0 || strings.ToLower(f) == "nkn")
+		if fromNKN {
+			break
+		}
+	}
+	if fromNKN && len(from) > 1 {
+		return nil, errors.New("should create only one source from NKN tunnel at one endpoint")
```

Comment:
Suggest "multiple tunnels is not supported when from NKN"

Reply:
Yes, modified.